File: python/two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for index, num in enumerate(nums):
            if (target - num) in nums:
                solution = set([index + 1, nums.index(target - num) + 1])
                if len(solution) == 1:
                    continue
                return sorted(solution)

File: rust/two-sum-ii-input-array-is-sorted.rs
impl Solution {
    pub fn two_sum(numbers: Vec<i32>, target: i32) -> Vec<i32> {
        for (index1, num) in numbers.iter().enumerate() {
            let diff = target - num;
            for index2 in (index1 + 1)..numbers.len() {
                println!("comparing {} & {}", numbers[index2], diff);
                if numbers[index2] == diff {
                    return Vec::from([(index1 + 1) as i32, (index2 + 1) as i32]);
                }
            }
        }
        return Vec::from([0 as i32, 1 as i32]);
    }
}
